lib/manageiq/release/repo.rb

```diff
@@ -54,7 +54,7 @@ def branch?(branch)
       private
 
       def git_clone
-        clone_source = options.clone_source || "git@github.com:ManageIQ/#{name}.git"
+        clone_source = options.clone_source || "git@github.com:#{github_repo}.git"
```

Comment:
We already have this logic in `github_repo` so it was simple to do and allowed me to test on the miq-test/sandbox.